filters/include/pcl/filters/frustum_culling.h

```diff
@@ -166,11 +166,16 @@ namespace pcl
       }
 
       /** \brief Set the near plane distance
-        * \param[in] np_dist the near plane distance
+        * \param[in] np_dist the near plane distance. Setting it to 0 to extract a square cone.
```

Comment:
```suggestion
        * \param[in] np_dist the near plane distance. You can set this to 0 to disable near-plane filtering and extract a pyramid instead of a frustum.
```

Reply:
Do we need to specify it's a rectangular pyramid?

Reply:
Doesn't hurt to add that